common/test/acceptance/pages/lms/discussion.py

```diff
@@ -720,6 +740,13 @@ def click_new_post_button(self):
             "New post action succeeded"
         ).fulfill()
 
+    def click_receive_update_checkbox(self):
+        """
+         Clicks the 'Receive Update' Checkbox.
+        """
+        receive_update_checkbox_element = self.q(css='.email-setting')
```

Comment:
Same question here as asked above. Why not a generic click method? That my be used to click any other element.